starknet_devnet/starknet_wrapper.py

```diff
@@ -755,24 +755,25 @@ async def mock_message_to_l2(self, transaction: InternalL1Handler) -> dict:
         return transaction.hash_value
 
     async def postman_flush(self) -> dict:
-        """Handles all pending L1 <> L2 messages and sends them to the other layer."""
+        """Handles all pending L1 <> L2 messages if such exists and sends them to the other layer."""
```

Comment:
I don't see these three new words as really necessary. In fact, the current wording is grammatically incorrect.

Reply:
rollbacked in 2eff0d05672cf00b5a63de88b6fc809ce5df205e